drivers/bluetooth/hci/ipc.c

```diff
@@ -265,19 +270,35 @@ static int bt_ipc_send(const struct device *dev, struct net_buf *buf)
 		break;
 	default:
 		LOG_ERR("Unknown type %u", bt_buf_get_type(buf));
+		err = -ENOMSG;
 		goto done;
 	}
 	net_buf_push_u8(buf, pkt_indicator);
 
 	LOG_HEXDUMP_DBG(buf->data, buf->len, "Final HCI buffer:");
-	err = ipc_service_send(&data->hci_ept, buf->data, buf->len);
+
+	for (retries = 0; retries < CONFIG_BT_HCI_IPC_SEND_RETRY_COUNT + 1; retries++) {
```

Comment:
`CONFIG_BT_HCI_IPC_SEND_RETRY_COUNT` could alternatively be called `CONFIG_BT_HCI_IPC_SEND_TRY_COUNT` and then have a minimum value of 1 to avoid the `+ 1`

Reply:
I think it would be bad. This would pose a threat of 0 value of tries. The +1 method allows for 0 retries and always 1 send call.

Reply:
> This would pose a threat of 0 value of tries.

That's why I suggested a minimum value of 1 in the Kconfig option. 

It's not a big deal :) It could also be of the form
```c
do {
    ...
} while (retries--);
```

But it's also OK as is; it's fairly obvious what's going on :)